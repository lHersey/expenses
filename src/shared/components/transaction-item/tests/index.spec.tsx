import { render } from '@testing-library/react-native';
import React from 'react';
import Providers from 'shared/providers/test-providers';

import TransactionItem from '../index';

describe('<TransactionItem />', () => {
  describe('snapshots', () => {
    it('should match snapshot', () => {
      expect.hasAssertions();
      const rendered = render(
        <Providers>
          <TransactionItem />
        </Providers>,
      ).toJSON();

      expect(rendered).toMatchSnapshot();
    });
  });
});
