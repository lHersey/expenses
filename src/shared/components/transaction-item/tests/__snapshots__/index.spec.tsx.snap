// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TransactionItem /> snapshots should match snapshot 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "overflow": "visible",
        },
        Object {
          "display": "flex",
          "flexDirection": "row",
          "justifyContent": "space-between",
        },
      ]
    }
  >
    <View
      style={
        Array [
          Object {
            "backgroundColor": "#FFF",
            "borderRadius": 15,
            "display": "flex",
            "flexGrow": 1,
            "overflow": "hidden",
            "paddingBottom": 15,
            "paddingLeft": 15,
            "paddingRight": 15,
            "paddingTop": 15,
          },
          Object {
            "display": "flex",
            "flexDirection": "row",
            "justifyContent": "space-between",
          },
        ]
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "flexDirection": "row",
            },
          ]
        }
      >
        <ViewManagerAdapter_ExpoLinearGradient
          proxiedProperties={
            Object {
              "colors": Array [
                4294944422,
                4294011498,
              ],
              "endPoint": undefined,
              "locations": undefined,
              "startPoint": undefined,
            }
          }
          style={
            Array [
              Object {
                "alignItems": "center",
                "borderRadius": 25,
                "height": 50,
                "justifyContent": "center",
                "marginRight": 10,
                "width": 50,
              },
            ]
          }
        >
          <Text />
        </ViewManagerAdapter_ExpoLinearGradient>
        <Text
          style={
            Array [
              Object {
                "color": "#3e3e3e",
                "fontSize": 16,
              },
              Object {
                "fontWeight": "bold",
              },
            ]
          }
          text="Food"
        >
          Food
        </Text>
      </View>
      <View
        style={
          Array [
            Object {
              "alignItems": "flex-end",
              "justifyContent": "space-around",
            },
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#3e3e3e",
                "fontSize": 16,
              },
              Object {
                "fontWeight": "500",
              },
            ]
          }
          text="$21.4"
        >
          $21.4
        </Text>
        <Text
          style={
            Array [
              Object {
                "color": "#3e3e3e",
                "fontSize": 16,
              },
              Object {
                "color": "#b0b0b0",
              },
            ]
          }
          text="Yesterday"
        >
          Yesterday
        </Text>
      </View>
    </View>
  </View>
</RNCSafeAreaProvider>
`;
